pkg/sources/github/github.go

```diff
@@ -1131,20 +1144,30 @@ var (
 )
 
 type repoInfo struct {
-	owner    string
-	repo     string
-	repoPath string
+	owner      string
+	repo       string
+	repoPath   string
+	visibility source_metadatapb.Visibility
 }
 
 func (s *Source) processRepoComments(ctx context.Context, repoPath string, trimmedURL []string, repoURL *url.URL, chunksChan chan *sources.Chunk) error {
+	if !(s.includeIssueComments || s.includePRComments) {
+		return nil
+	}
```

Comment:
Is the idea here that the issue or PR initial description constitutes a "comment" from a configuration standpoint?

Reply:
The existing code implies such, which I think makes sense. 

This guard will exit early and prevent needless calls to `s.visibilityOf`. Alternatively, we could wrap the [`s.processRepoComments` call](https://github.com/trufflesecurity/trufflehog/blob/20eb89d089095425c0feae01f78ddadb400c2839/pkg/sources/github/github.go#L1088) in this condition.